### PR TITLE
Adding samtools version 1.3 in a new directory

### LINKED ISSUE #77: Update samtools to version 1.3
Samtools version 1.3 release notes
https://github.com/samtools/samtools/releases/tag/1.3

### PR DESCRIPTION
**Pull request by heuermh**
Fixes #77 

Alternative to #78

### CODE REVIEW

File: samtools-1.3/Makefile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+build_tool = runtime-container.DONE
+git_commit ?= $(shell git log --pretty=oneline -n 1 -- ../samtools-1.3 | cut -f1 -d " ")
+name = quay.io/ucsc_cgl/samtools
```
Comment (heuermh) [inline samtools-1.3/Makefile:3]: @hannes-ucsc when in a separate directory, should the version also be in the name?

Comment (hannes-ucsc) [inline samtools-1.3/Makefile:3]: Judging from ubu and mapsplice, I think the image name should be just "samtools" and the version should be in the tag.

Comment (hannes-ucsc) [inline samtools-1.3/Makefile:3]: https://quay.io/repository/ucsc_cgl/ubu?tab=tags

Comment (hannes-ucsc) [inline samtools-1.3/Makefile:3]: And that's what you are currently doing, so I merged it as is.

**Comment by heuermh** [comment]
Thanks!

**Comment by hannes-ucsc** [comment]
https://quay.io/repository/ucsc_cgl/samtools?tab=tags
1.3--323f40b0c5dace98ced5521c6ae45a6c04df923b

**Pull request merged**